packages/query/src/stacks/balance/use-stx-balance.query.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { describe, expect, it, vi } from 'vitest';
+
+import { createMoney } from '@leather.io/utils';
+
+import { useStxBalanceQuery } from './use-stx-balance.query';
+
+vi.mock('../stacks-client', () => ({
+  useStacksClient: vi.fn(),
+}));
+
+vi.mock('../../leather-query-provider', () => ({
+  useCurrentNetworkState: vi.fn().mockReturnValue({
+    chain: {
+      stacks: {
+        url: 'https://api.mainnet.hiro.so',
+      },
+    },
+  }),
+}));
+
+// Mock the query hook response
+vi.mock('@tanstack/react-query', () => ({
+  useQuery: () => ({
+    data: createMoney(1000000, 'STX'),
+    isLoading: false,
+    error: null,
+  }),
+}));
+
```

Comment:
I wanted to add some coverage here for the stats:

![Screenshot 2024-10-31 at 12 39 18](https://github.com/user-attachments/assets/66191f3c-195b-44e2-aba3-0105001e1501)

But this test seems a bit pointless. I am just mocking `vi.mock('@tanstack/react-query'` so it's kind of just testing the mocking works when called through the hook. 

Mentioning as I'd like if we could add coverage to these hooks but not sure if it's superfluous